### ISSUE TITLE
Issue upgrading to 0.1.3

### ISSUE THREAD

**Issue by Tang8330** [rajinwonderland/react-code-blocks#95]
```
TS2322: Type '{ text: string; language: string; showLineNumbers: boolean; startingLineNumber: number; theme: { lineNumberColor: string; lineNumberBgColor: string; backgroundColor: string; textColor: string; ... 29 more ...; numberColor: string; }; wrapLines: true; }' is not assignable to type 'IntrinsicAttributes & ExecutionProps & RefAttributes<({ theme, text, codeBlock, customStyle, onCopy, ...rest }: CopyBlockProps) => React.JSX.Element>'.
  Property 'text' does not exist on type 'IntrinsicAttributes & ExecutionProps & RefAttributes<({ theme, text, codeBlock, customStyle, onCopy, ...rest }: CopyBlockProps) => React.JSX.Element>'.
    473 |         </p>
    474 |         <CopyBlock
  > 475 |           text={SETTINGS["postgres_heartbeat_table"]}
        |           ^^^^
    476 |           language={"sql"}
    477 |           showLineNumbers={false}
    478 |           startingLineNumber={1}
```

**Comment by thomasmost** [comment]
closed in #96

**Comment by samuelvisscher** [comment]
@rajinwonderland I have the same issue as @Tang8330 , but I don't understand why we have this unusual syntax of filling out props?

Why would this be correct:

```
                <CodeBlock
                  {...{
                    language: "text",
                    showLineNumbers: true,
                    text: "hello",
                    theme: dracula,
                  }}
                />
```

And this incorrect:

```
                <CodeBlock
                  language="text"
                  showLineNumbers={true}
                  text="hello"
                  theme={dracula}
                />
```
